azureenergylabelerlib/entities.py

```diff
@@ -724,16 +724,17 @@ def _export_to_fs(self, directory, filename, data):
     def _export_to_blob(self, blob_url, filename, data):
         """Exports as json to Blob container object storage."""
         parsed_url = urlparse(blob_url)
-        blob_service_client = BlobServiceClient(account_url=f'{parsed_url.scheme}://{parsed_url.netloc}/',
+        # This enables uploads using URLs that include a SAS token, which is the safest
+        blob_service_client = BlobServiceClient(account_url=blob_url) if parsed_url.query else BlobServiceClient(account_url=f'{parsed_url.scheme}://{parsed_url.netloc}/',
                                                 credential=self._credentials)
         container = parsed_url.path.split('/')[1]
         blob_client = blob_service_client.get_blob_client(container=container, blob=filename)
         try:
             blob_client.upload_blob(data.encode('utf-8'), overwrite=True)
-            self._logger.info(f'File {filename} copied to blob {blob_url}')
+            self._logger.info(f'File {filename} copied to blob {parsed_url.scheme}://{parsed_url.netloc}{parsed_url.path}')
         except Exception as error:  # pylint: disable=broad-except
             self._logger.error(error)
-            self._logger.info(f'File {filename} copy to blob {blob_url} failed')
+            self._logger.info(f'File {filename} copy to blob {parsed_url.scheme}://{parsed_url.netloc}{parsed_url.path} failed')
```

Comment:
Should this maybe be a wanting level? If a file failed to upload i guess it is not just info.